arcgis-tiledlayer-rendermode/src/main/java/com/esri/arcgisruntime/sample/arcgistiledlayerrendermode/MainActivity.java

```diff
@@ -108,99 +108,99 @@ public void onNothingSelected(AdapterView<?> parent) {
         switch (progress) {
           case 0:
             mMapView.setViewpointScaleAsync(5.91657527591555E8);
-            mSeekBarScale.setText("Set Scale - "+ 5.91657527591555E8);
+            mSeekBarScale.setText("Seekbar Scale - "+ 5.91657527591555E8);
```

Comment:
This applies to all the `setText()` methods below:  Can be simplified using string resources with placeholders instead of concatenating strings displayed with `setText()`.  

String resource:
```xml
<string name="seekbar_scale">Seekbar Scale - %1$d</string>
```

usage
```java
getString(R.string.seekbar_scale, 5.91657527591555E8);
```